compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -11536,7 +11533,7 @@ private STNode parseTableConstructorOrQueryRhs(STNode tableKeyword, STNode keySp
     /**
      * Parse query construct type.
      * <p>
-     * <code>query-construct-type := table key-specifier | stream</code>
+     * <code>query-construct-type := table key-specifier | stream | map</code>
```

Comment:
Shall we update the doc in the parent function as well
https://github.com/ballerina-platform/ballerina-lang/blob/f31ad7af45409bd9a68a3e6ad36b58286607ab35/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java#L11469